server/public/styles.css

```diff
@@ -0,0 +1,188 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family:
+    'Press Start 2P',
+    cursive;
+}
+
+body {
+  background-color: rgb(44, 51, 59);
+}
+
+.navbar {
+  background-color: rgb(89, 202, 161);
+  height: 70px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.name-logo {
+  justify-self: center;
+  color: white;
+  width: 100px;
+}
+
+.fa-bars {
+  color: white;
+}
+
+.fa-times {
+  color: white;
+  font-size: 2rem;
+}
```

Comment:
Instead of adding styles to 3rd-party classes, just add your own custom classes.